src/zcbenchmarks.cpp

```diff
@@ -31,6 +31,31 @@
 #include "zcash/IncrementalMerkleTree.hpp"
 
 using namespace libzcash;
+// This method is based on Shutdown from init.cpp
+void shutdown_wallet()
+{
+    LogPrintf("%s: In progress...\n", __func__);
+    static CCriticalSection cs_Shutdown;
+    TRY_LOCK(cs_Shutdown, lockShutdown);
+    if (!lockShutdown)
+        return;
```

Comment:
Hmm, this is probably good to have (so we don't conflict with the node actually being shut down), but as this is in a separate method and not inside the benchmark, we need to ensure we actually return from the benchmark itself. Probably would be best to throw an error here.

Reply:
done